README.md

```diff
@@ -98,15 +98,123 @@ This release extends support for RHEL v8.5.
 
 ### Supported GPUs
 
-#### Radeon Pro V620 and W6800 Workstation GPUs
+Support can be assumed for current server accelerators GPUs intended for enterprise customers.
+This ROCm release officially supports workstation GPU products.
+However, there are also compatible consumer GPU products available.
+Finally, this ROCm release can be used with a number of older GPU products.
 
-This release extends ROCm support for Radeon Pro V620 and W6800 Workstation GPUs.
+#### Enterprise GPUs
 
-- SRIOV virtualization support for Radeon Pro V620
-- KVM Hypervisor (1VF support only) on Ubuntu Host OS with Ubuntu, CentOs, and RHEL Guest
-- Support for ROCm-SMI in an SRIOV environment. For more details, refer to the ROCm SMI API documentation.
+Current enterprise GPU accelerators can be assumed to be supported. These include:
+
+- AMD Instinct™ MI100
+- AMD Instinct™ MI200
+- AMD Instinct™ MI250
+- AMD Instinct™ MI250X
+
+#### Supported Workstation GPUs
+
+This ROCm release extends support to two workstation GPU products:
```

Comment:
Does this means that there exist some other workstation GPU products which were supported before and are still supported by this release?

Reply:
Yes, I see your point. I am not sure why this sentence talks about *extending* support, either. I am not aware of any workstation GPUs being officially supported other than the 6800W and the V620. Maybe this choice of words doesn't mean anything, but I still tried to stay as close to it, as I could in the interest of making this the most acceptable contribution it can be.